src/input.rs

```diff
@@ -25,23 +25,25 @@ use std::{
 /// Here's a non-exhaustive list of the most commonly used types to get you started:
 ///
 /// - [`String`] and [`&str`],
-/// - [`Split`] (and its shortcut `%`),
+/// - [`Split`] (and its shortcut `%`) to split commands by whitespace,
 /// - [`PathBuf`] and [`&Path`],
 /// - multiple sequence types, like [`vectors`], [`slices`] and (since version 1.51) [`arrays`],
 /// - [`CurrentDir`],
+/// - [`SetVar`] for setting environment variables,
 /// - [`StdIn`], and
 /// - [`LogCommand`].
 ///
 /// [`String`]: trait.Input.html#impl-Input-for-String
 /// [`&str`]: trait.Input.html#impl-Input-for-%26str
-/// [`Split`]: trait.Input.html#impl-Input-3
+/// [`Split`]: trait.Input.html#impl-Input-4
 /// [`PathBuf`]: trait.Input.html#impl-Input-for-PathBuf
 /// [`&Path`]: trait.Input.html#impl-Input-for-%26Path
 /// [`vectors`]: trait.Input.html#impl-Input-for-Vec<T>
 /// [`slices`]: trait.Input.html#impl-Input-for-%26[T]
 /// [`arrays`]: trait.Input.html#impl-Input-for-[T%3B%20N]
-/// [`CurrentDir`]: trait.Input.html#impl-Input-1
-/// [`StdIn`]: trait.Input.html#impl-Input-2
+/// [`CurrentDir`]: trait.Input.html#impl-Input-2
+/// [`SetVar`]: trait.Input.html#impl-Input-1
+/// [`StdIn`]: trait.Input.html#impl-Input-3
```

Comment:
Unfortunately these links are not stable when adding impls. I don't want to fix this in this PR, but I opened this issue instead: #111.